app/actions/mentors.js

```diff
@@ -0,0 +1,24 @@
+import { createAction } from "redux-actions";
+
+import {
+  SET_MENTORS,
+  CLEAR_MENTORS,
+} from "action-types";
+
+import { mentors } from "lib/api";
+
+const setMentors = createAction(SET_MENTORS);
+
+export const getMentors = (filters = {}) => {
+  return dispatch => {
+    return mentors
+      .index(filters)
+      .then(response => {
+        dispatch(setMentors(response.data));
+        return Promise.resolve(response.data);
```

Comment:
can't you just `return response.data`? (The same applies to all instances of this pattern below)